ctru-rs/src/console.rs

```diff
@@ -38,6 +39,18 @@ pub enum Dimension {
     Height,
 }
 
+/// Destination for stderr redirection with [`redirect_stderr`].
+#[doc(alias = "debugDevice")]
+#[repr(u32)]
+pub enum Destination {
```

Comment:
I'm not sure if we should call this `Destination` or `Redirect` or what. I'm open to suggestions about that.